src/index.js

```diff
@@ -85,48 +85,58 @@ const GravityFormForm = ({ id, formData, lambda, presetValues = {} }) => {
-                    }
-                    id={`gravityform--id-${id}`}
-                    key={`gravityform--id-${id}`}
-                    onSubmit={handleSubmit(onSubmitCallback)}
-                >
-                    {generalError && (
-                        <FormGeneralError errorCode={generalError} />
-                    )}
-
-                    <FieldBuilder
-                        errors={errors}
-                        formData={singleForm}
-                        formId={id}
-                        presetValues={presetValues}
-                        register={register}
-                    />
-                    <button className="gravityform__button" type="submit">
-                        {singleForm.button.text
-                            ? singleForm.button.text
-                            : 'Submit'}{' '}
-                        {formLoading && (
-                            <span className="gravityform__button__loading_span">
-                                Loading
-                            </span>
+            <div className="gform_wrapper" id={`gform_wrapper_${id}`}>
+                <div className="gform_anchor" id={`gf_${id}`} />
+                {singleForm && (
+                    <form
+                        className={
+                            formLoading
+                                ? `gravityform gravityform--loading gravityform--id-${id}`
+                                : `gravityform gravityform--id-${id}`
+                        }
+                        //TODO: ID change go GF standard "gfrom_1"?
+                        id={`gravityform--id-${id}`}
+                        key={`gravityform--id-${id}`}
+                        onSubmit={handleSubmit(onSubmitCallback)}
+                    >
+                        {generalError && (
+                            <FormGeneralError errorCode={generalError} />
                         )}
-                    </button>
-                </form>
-            )
+                        <div className="gform_body">
+                            <ul
+                                className={`gform_fields form_sublabel_${singleForm.subLabelPlacement} description_${singleForm.descriptionPlacement} ${singleForm.labelPlacement}`}
+                                id={`gform_fields_${id}`}
+                            >
+                                <FieldBuilder
+                                    errors={errors}
+                                    formData={singleForm}
+                                    formId={id}
+                                    presetValues={presetValues}
+                                    register={register}
+                                />
+                            </ul>
+                        </div>
+
+                        <div
+                            className={`gform_footer ${singleForm.labelPlacement}`}
+                        >
+                            <input
```

Comment:
What is the reason for changing from button to input? Button is used to show a change on the page without redirecting?

Reply:
Because that's how GF renders it, but I'll take some time to see if it really matters for the CSS rules.

Reply:
It doesn't. All good :) When I use GF with WordPress I have a custom function which changes it from an input to a button. No issues.